.cproject

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
put the files that eclipse generates in .gitignore please :) they are platform specific for most part and not really in the scope.